build.gradle

```diff
@@ -57,7 +57,7 @@ configure(subprojects) {
         junitVersion = '4.12'
         jupiterVersion = '5.3.2'
         kotlinVersion = '1.3.41'
-        knowmXchangeVersion = '4.3.3'
+        knowmXchangeVersion = '4.4.2'
```

Comment:
This is the library that abstract away the different exchange integrations. Since we want to have the most up-to-date integration possible (e.g. broadest set of exchanges, support for most recent exchange API versions, etc) it makes sense to use the latest version of this lib.

Reply:
This is the library that abstract away the different exchange integrations. Since we want to have the most up-to-date integration possible (e.g. broadest set of exchanges, support for most recent exchange API versions, etc) it makes sense to use the latest version of this lib.